src/ImportDefinitionsBundle/ProcessManager/ImportDefinitionsReport.php

```diff
@@ -155,6 +163,24 @@ protected function processLine($line, &$result)
         $this->processChecks($line, $result);
     }
 
+    /**
+     * @param $line
+     * @param $result
+     * @return bool
+     */
+    protected function checkForArtifact($line, &$result)
```

Comment:
This is the wrong approach, it would be better to have an event dispatcher monitor the logger and dispatch events based on that.